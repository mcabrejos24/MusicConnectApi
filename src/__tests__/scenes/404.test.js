import React from "react";
import { shallow } from "enzyme";
import FourOhFour from '../../scenes/404';
import toJson from "enzyme-to-json";

describe("renders 404", () => {
    let page;
    beforeEach(() => {
        page = shallow(<FourOhFour />);
    });
    it("404 loads", () => {
        page;
    });
    it("props class name matches", () => {
        expect(page.props().className).toEqual("content four-oh-four-page");
    });
    it("matches snapshot", () => {
        expect(toJson(page)).toMatchSnapshot();
    });
});
