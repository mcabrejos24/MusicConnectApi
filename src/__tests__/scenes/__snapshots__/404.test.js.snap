// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 404 matches snapshot 1`] = `
<div
  className="content four-oh-four-page"
>
  404
</div>
`;
